### ISSUE TITLE
Audio could not be played entirely on iOS/macOS

### ISSUE THREAD

**Issue by dumganhar** [cocos2d/cocos2d-x#16938]
Reported at http://forum.cocos.com/t/topic/41723

- cocos2d-x version: v3.14
- devices test on: iOS, macOS
- developing environments
   - NDK version: N/A
   - Xcode version: 8.1
   - VS version: N/A
   - browser type and version:N/A

Steps to Reproduce:

Just play this audio [1018.mp3.zip](https://github.com/cocos2d/cocos2d-x/files/621300/1018.mp3.zip).

There are some frames at the end could not be played.
It seems that audio automatically stops before it's finished.

**Comment by dumganhar** [comment]
```c++
        // Get the total frame count
        propertySize = sizeof(totalFrames);
        error = ExtAudioFileGetProperty(extRef, kExtAudioFileProperty_FileLengthFrames, &propertySize, &totalFrames);
```

returns wrong frame size, it should be **61440**, but it returns **50688**.

**Comment by dumganhar** [comment]
fixed at #16946

**Comment by dumganhar** [comment]
Closed this issue since https://github.com/cocos2d/cocos2d-x/pull/16946 was merged.
